airflow/sensors/base.py

```diff
@@ -114,15 +118,33 @@ def _orig_start_date(
     )
 
 
+class SkipPolicy(str, enum.Enum):
```

Comment:
I feel we might need to update documentation for this as well

Reply:
yes I will in another PR , when we agree on the implementation , cause I don't want to do big refacto every time we change the terminology